todo_app/templates/index.html

```diff
@@ -11,8 +11,19 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items %}
+          <li>{{item.title}}
+            <form class="d-inline-flex form-inline" action="/item/{{item.id}}/delete" method="post">
+              <input type="submit" value="Delete"/>
+            </form>
+          </li>
+        {% endfor %}
       </ul>
+      <form action="/item" method="post">
```

Comment:
You could use `url_for("add_item")` here instead.

Also, `post` is the default method for a form, so could be removed.

Reply:
Sorry - I was wrong!